tests/README.md

```diff
@@ -0,0 +1,202 @@
+# Cloudbeat Automated Tests Framework (C-ATF)
+> 
+
+This project provides an ability to develop component and integration tests for Cloudbeat.
+
+
+## Getting started
+
+All installations below (except of docker desktop) are available via [Homebrew](https://brew.sh/).
```

Comment:
missing installation for python

Reply:
Poetry package manager installs python if not installed yet.